sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -52,98 +53,116 @@
 class Options(interactive_options.InteractiveOptions):
   """Options that guide how Interactive Beam works."""
   @property
-  def enable_capture_replay(self):
-    """Whether replayable source data capture should be replayed for multiple
-    PCollection evaluations and pipeline runs as long as the data captured is
+  def enable_record_replay(self):
```

Comment:
enable_recording_replay. Basically, if "capture" is used as a noun, change it to "recording" instead of "record", since "record" might have a notion of an individual record. Please check all occurrences.